shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/network/core/NetworkClient.kt

```diff
@@ -55,6 +55,13 @@ open class NetworkClient {
         }
     }
 
+    inline fun fetchEmpty(builder: HttpRequestBuilder) : Flow<Unit> {
```

Comment:
@blyscuit Just curious, may I know why we need this function? 🤔

Reply:
The original `fetch` will try to map JSON and returns error if unable to map (including  empty JSON). We are using this function to explicitly skip the JSON mapping.